uberfire-extensions/uberfire-wires/uberfire-wires-core/uberfire-wires-core-grids/src/main/java/org/uberfire/ext/wires/core/grids/client/model/GridData.java

```diff
@@ -295,6 +295,39 @@ void collapseCell(final int rowIndex,
     void expandCell(final int rowIndex,
                     final int columnIndex);
 
+    /**
+     * Updates the width of columns with {@link GridColumn.ColumnWidthMode#AUTO}
+     * @return a boolean that indicates if grid need to be redraw or not
+     */
+    boolean refreshWidth();
+
+    /**
+     * Updates the width of columns with {@link GridColumn.ColumnWidthMode#AUTO}
+     * @param currentWidth is the grid width before this resize iteration
+     * @return a boolean that indicates if grid need to be redraw or not
+     */
+    boolean refreshWidth(double currentWidth);
```

Comment:
I think it could be useful to have as public method not only the one without parameter but also this one. I tried to think about another general use case too